lib/game.rb

```diff
@@ -0,0 +1,32 @@
+class Game
+  attr_reader :comp_move
+
+  def generatemove()
+    moves =["rock","paper","scissors"]
+    num = rand(3)
+    return moves[num]
+  end
```

Comment:
Rand is a great method. Another way of doing this might be ['rock', 'paper', 'scissors'].sample